frappe/public/js/frappe/form/controls/text_editor.js

```diff
@@ -1,5 +1,28 @@
 import Quill from 'quill';
 
+// specify the fonts you want
+let fonts = ['Arial', 'Courier', 'Times New Roman', 'Verdana'];
+// generate code friendly names
+function getFontName(font) {
```

Comment:
Also, we use [snake_case](https://en.wikipedia.org/wiki/Snake_case) for naming variables and functions.
Please replace
```
fontNames -> font_names
fontStyles -> font_styles
getFontName() -> get_font_name()
```

Reply:
Notes have been fixed.